staging/src/k8s.io/kube-aggregator/pkg/apiserver/patch_always_local_delegate.go

```diff
@@ -21,7 +21,18 @@ func AddAlwaysLocalDelegateForPrefix(prefix string) {
 	alwaysLocalDelegatePathPrefixes.Insert(prefix)
 }
 
+var overlappingGroupVersion = map[schema.GroupVersion]bool{}
+
+// AddOverlappingGroupVersion will stop the CRD registration controller from trying to manage an APIService.
+func AddOverlappingGroupVersion(groupVersion schema.GroupVersion) {
```

Comment:
how many carry commits are around this topic of APIService fight between CRDs and aggregation? Can we squash those?